data/campaigns/emp04.wmf/scripting/tribes/brewery1/init.lua

```diff
@@ -53,7 +53,7 @@ descriptions:new_productionsite_type {
 
    programs = {
       main = {
-         descname = "brewing beer",
+         descname = pgettext("empire_building", "brewing beer"),
```

Comment:
this is a name of a working program --> we should use _

Reply:
Oh, I see. Well, I wasn't sure about using the empire_building context for them, but all strings seemed to use it, like e.g. "No soldier to train!" in trainingcamp1.